.github/workflows/workflows-matrix.json

```diff
@@ -0,0 +1,7 @@
+[
+  "functions/concepts",
+  "functions/helloworld",
```

Comment:
I'd be tempted to pull this and generate.js into a utils directory, or config directory? So that it's clear that these aren't workflows themselves, they're just used to generate workflows.